analytics/lib/fixtures.py

```diff
@@ -62,11 +62,11 @@ def generate_time_series_data(
         seasonality[i % len(seasonality)] * (growth_base**i) for i in range(length)
     ]
 
-    seed(random_seed)
```

Comment:
This function has some other calls to `random()` above this line. It’s not clear whether the intention was for those to be determined by this seed, but currently they are not…

Reply:
I can't imagine that being intentional.

Reply:
Moved. That will have the effect of changing the exact results for a given `random_seed`, but it doesn’t seem to break the tests so it’s probably fine.